wpilibjIntegrationTests/src/main/java/edu/wpi/first/wpilibj/PDPTest.java

```diff
@@ -57,32 +54,33 @@ public PDPTest(MotorEncoderFixture<?> mef, Double expectedCurrentDraw) {
     m_expectedStoppedCurrentDraw = expectedCurrentDraw;
   }
 
-  @Parameters(name = "{index}: {0}, Expected Stopped Current Draw: {1}")
   public static Collection<Object[]> generateData() {
     // logger.fine("Loading the MotorList");
     return Arrays.asList(new Object[][] {{TestBench.getTalonPair(), 0.0}});
   }
 
-  @After
+  @AfterEach
   public void tearDown() {
     me.reset();
   }
 
   /** Test if the current changes when the motor is driven using a talon. */
-  @Test
+  @ParameterizedTest(name = "{index}: {0}, Expected Stopped Current Draw: {1}")
+  @MethodSource("generateData")
   public void checkStoppedCurrentForMotorController() throws CANMessageNotFoundException {
```

Comment:
`@ParameterizedTest` without parameters?

Reply:
I haven't ported all the parameterized tests to JUnit 5 yet. JUnit 4 declared the parameters in one spot instead of at each function.